ipn/ipnlocal/local.go

```diff
@@ -219,6 +219,14 @@ type LocalBackend struct {
 	serveListeners     map[netip.AddrPort]*serveListener // addrPort => serveListener
 	serveProxyHandlers sync.Map                          // string (HTTPHandler.Proxy) => *httputil.ReverseProxy
 
+	// getTCPHandlerForFunnelFlow conditionally handles an incoming TCP flow for
+	// the provided src and dport. It returns a handler function to handle the
+	// flow, and returns nil if the flow should be handled by serve listeners.
```

Comment:
I don't quite follow this bit about nil.

Reply:
Reworded, ptal